tiledb/common/algorithm/test/unit_parallel_merge.cc

```diff
@@ -35,11 +35,21 @@
 #include <test/support/tdb_catch.h>
 #include <test/support/tdb_rapidcheck.h>
 
+#if defined(TILEDB_INTERCEPTS)
```

Comment:
We are already compiling a test-only file, so this test-only define is not needed.